app/proguard-rules.pro

```diff
@@ -55,15 +55,13 @@
     public static *** v(...);
     public static *** i(...);
 }
+-assumenosideeffects class java.lang.String {
+    public static java.lang.String format(...);
```

Comment:
This is to get rid of lonely `String.format`s in the bytecode after the log statements have been stripped out.

Reply:
You would expect Proguard could figure it out